storage-backends/elasticsearch/src/main/java/com/expedia/alertmanager/store/backend/Writer.java

```diff
@@ -76,7 +76,8 @@ public void onResponse(BulkResponse bulkItemResponses) {
                 retry(new RuntimeException("Fail to execute the elastic search write with partial failures:"
                         + bulkItemResponses.buildFailureMessage()));
             } else {
-                callback.onComplete(null);
+                callback.onComplete(new Exception("None"));
```

Comment:
may be a better option is to use Optionals - `Optional.empty()` and do an empty check in the method instead of null check via `ex.ifPresent(e -> Assert.fail(e.getMessage()));`
You will have to change the signature of the `onComplete` method to accept `Optional<Exception>`

Reply:
yeah. that sounds a better approach. @dinilatgit